app/routes/circulars._index.tsx

```diff
@@ -174,6 +179,73 @@ function Pagination({
   )
 }
 
+function DownloadModal() {
+  const modalRef = useRef<ModalRef>(null)
+  return (
+    <>
+      <h2>Advanced</h2>
+      <ModalToggleButton
+        modalRef={modalRef}
+        opener
+        outline
+        className="text-middle"
+      >
+        <Icon.FileDownload className="bottom-aligned margin-right-05" />
+        Download Archive
+      </ModalToggleButton>
+      <Modal
+        renderToPortal={false}
+        ref={modalRef}
+        id="example-modal-1"
+        aria-labelledby="modal-1-heading"
+        aria-describedby="modal-1-description"
+      >
+        <ModalHeading id="modal-1-heading">
+          GCN Circulars Database Download
+        </ModalHeading>
+        <div className="usa-prose">
+          <p id="modal-1-description">
+            This is a download of the entire GCN Circulars databse.
+          </p>
+          <p>It may take a moment.</p>
+          <p>Select a file format to begin download.</p>
+        </div>
+        <ModalFooter>
+          <ButtonGroup>
+            <ModalToggleButton modalRef={modalRef} closer>
+              <a
+                className="text-no-underline text-white"
+                href={`/circulars/archive.txt.tar`}
+                download={'archive.tar.gz'}
```

Comment:
```suggestion
```

Reply:
<img width="1303" alt="Screen Shot 2023-08-24 at 9 19 38 AM" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/19705410-8439-4c76-8d67-c89e8a40cdc2">
This is what happens if you take the download portion out of the a tag.

Reply:
Ah! That's because you aren't setting the MIME type of the response in your two new loaders. Please set the MIME type and then try removing the `download` attribute.